packages/xcm-util/asset-sort/compare-assets.ts

```diff
@@ -437,7 +437,7 @@ export function compareAnyAssetId(
   assetId1: AnyAssetId,
   assetId2: AnyAssetId,
 ) {
-  if (xcmVersion === 4) {
+  if (xcmVersion === 4 || xcmVersion === 5) {
```

Comment:
Please take extra care when handling V5 NetworkIds. Are there no issues with that?